resource-managers/kubernetes/core/src/main/scala/org/apache/spark/deploy/k8s/KubernetesUtils.scala

```diff
@@ -381,4 +381,34 @@ object KubernetesUtils extends Logging {
       }
     }
   }
+
+  /**
+   * This function builds the EnvVar objects for each key-value env.
+   */
+  @Since("3.4.0")
+  def buildEnvVars(env: Map[String, String]): Seq[EnvVar] = {
+      env.filter( env => env._2 != null)
```

Comment:
Apache Spark uses 2-space indentation. Could you adjust this?

Reply:
Also, `( env =>` -> `(env =>`

Reply:
Yes, updated